<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>980</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Account Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>static/icon.png</normaloff>static/icon.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LeftMainMenu">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="UserInitialization">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>140</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="title">
        <string>User initialization</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QTextBrowser" name="UserStatus">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>20</y>
          <width>161</width>
          <height>111</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8.25pt;&quot;&gt;Sign in to your account.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>201</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="MainButtons">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_Login">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="Line_Login">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="Line_Password">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_lPassword">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="Btn_Login">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Log in</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="btn_AddNew">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Add new user</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="AddNewAccount">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>180</height>
        </size>
       </property>
       <property name="title">
        <string>Add new account</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>371</width>
          <height>155</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="main_layout">
         <item row="2" column="1">
          <widget class="QLineEdit" name="line_login">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>LeusNazarii@gmail.com</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="line_password">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>529984073_Nazar1998</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_login">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Login*</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lbl_password">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Password*</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QPushButton" name="btn_AddNewAccount">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lbl_Email">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_pswd_to">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Password to*</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_pswd_to">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>worldoftanks.ru/</string>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="line_Email">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>LeusNazarii@gmail.com</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lbl_username">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>User name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="line_username">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Carpe_Diem</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="btn_generate_pswd">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Generate a strong password</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>* -  required fields</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="label_add_info">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="GenerateStrongPassword">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Generate strong password</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>371</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="mainLayout">
         <item row="1" column="1">
          <widget class="QLabel" name="label_ValidCharacters">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>All valid characters (recomended):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="symbols_amount">
           <property name="accelerated">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>6</number>
           </property>
           <property name="maximum">
            <number>25</number>
           </property>
           <property name="value">
            <number>14</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="radioButton_default_symbols">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="toolTipDuration">
            <number>-1</number>
           </property>
           <property name="text">
            <string>A-Z, a-z, 0-9, _</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Symbols amount(6-25):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="checkBox_A_Z">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>A-Z</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="checkBox_0_9">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>0-9</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="checkBox_a_z">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>a-z</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="checkBox_Sym">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Symbol: _</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="pushButton_Generate_pswd">
           <property name="text">
            <string>Generate password</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="lineEdit_Ready_Pass">
           <property name="text">
            <string>UUjsdf_Difj3</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>561</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="TableParamLayout">
     <item>
      <layout class="QVBoxLayout" name="TPL_Main">
       <item>
        <widget class="QGroupBox" name="TBL_group_box">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Table parameters</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>541</width>
            <height>94</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="ButtonsLayout">
           <item row="0" column="1">
            <widget class="QCheckBox" name="ShowPswrd">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Show passwords</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="ShowLogins">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Show logins</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="ShowEmails">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Show Emails</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="ShowDate">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Show date of last changes</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="ShowUsrNames">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Show user names</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btn_Apply">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>561</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="TableLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>12</number>
       </property>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Password to</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Password</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>User name</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Email</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Last modified date</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
